<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PIAGUIClass</class>
 <widget class="QMainWindow" name="PIAGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1026</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PIAGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyOpenGLWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>-1</x>
      <y>-1</y>
      <width>671</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>420</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PIA</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="CtlPoints">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Control Vertices</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="CtlPoly">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>50</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Control Polygon</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="CurvaturePole">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>110</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Curvature Pole</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="BsplineCurve">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>B-spline Curve</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="OneStepIterate">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>520</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>One Step Iterate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>10</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Display Option</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>440</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LSPIA</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>470</y>
      <width>89</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TargetError">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>160</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0.01</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>200</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Fitting Error</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>160</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Target Error</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FittingError">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>200</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>9999999</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DataPoints">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Data Vertices</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1026</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="OpenFile"/>
   </widget>
   <widget class="QMenu" name="data">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actiongcur"/>
    <addaction name="actionLSPIA"/>
    <addaction name="actionSPARSE"/>
   </widget>
   <addaction name="file"/>
   <addaction name="data"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="OpenFile">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actiongcur">
   <property name="text">
    <string>g.cur</string>
   </property>
  </action>
  <action name="actionLSPIA">
   <property name="text">
    <string>LSPIA</string>
   </property>
  </action>
  <action name="actionSPARSE">
   <property name="text">
    <string>SPARSE</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QWidget</extends>
   <header>myopenglwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PIAGUI.qrc"/>
 </resources>
 <connections/>
</ui>
